### ISSUE TITLE
Legionella bacteria in sputum

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10988]
**Preferred term label:**
Legionella bacteria growth in sputum culture

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Legionnaires' disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001218
- label: Positive Legionella sputum culture
- parents: [Sputum infectious agent (HP:0430107)](https://hpo.jax.org/app/browse/term/HP:0430107)
- pmid: PMID:38501659
- def: Growth of Legionella bacteria in a sputum culture.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10988
- synonyms:
